src/platform/intel/ace/lib/pm_runtime.c

```diff
@@ -52,6 +54,8 @@ const struct pm_state_info *pm_policy_next_state(uint8_t cpu, int32_t ticks)
 			/* TODO: PM_STATE_RUNTIME_IDLE requires substates to be defined
 			 * to handle case with enabled PG andf disabled CG.
 			 */
+			tr_info(&power_tr, "transition to state %x (min_residency = %u, exit_latency = %u)",
```

Comment:
Once confirmed this log functionality, could change it to tr_dbg? log print always take a lot of cycles, especially in core enabling and ipc message handling, even one line log could cause ipc timeout.

Reply:
ACK

Reply:
Nice to have %#x for hex